include/infinispan/hotrod/NearCacheConfiguration.h

```diff
@@ -14,7 +14,20 @@
 namespace infinispan {
 namespace hotrod {
 
-enum NearCacheMode { DISABLED=0, INVALIDATED=1 };
+/**
+ * Enumeration of all the near cache mode of operation
+ */
+enum NearCacheMode {
+    DISABLED=0,     /*!< No near cache. */
+    INVALIDATED=1   /*!< Near cache in invalidated mode. In invalidated mode an entry is create near only after a get
```

Comment:
In invalidated mode an entry is create near-> In invalidated mode an entry in the near cache is created